### PR TITLE
New yasmallbbob

### PR DESCRIPTION
**Pull request by fteytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Adding point 2 in #466 
In comparison to yabbob, I have removed algorithms which seem to be non relevant in that context (namely cobyla).
 

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/benchmark/experiments.py [hunk 1]
```diff
@@ -246,15 +246,19 @@ def paramultimodal(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
...
-    optims = ["NaiveTBPSA", "TBPSA", "NGO", "Shiva", "DiagonalCMA", "CMA", "PSO", "DE", "MiniDE", "QrDE", "MiniQrDE", "LhsDE", "OnePlusOne",
+    optims = ["NaiveTBPSA", "TBPSA", "DiagonalCMA", "CMA", "PSO", "DE", "MiniDE", "QrDE", "MiniQrDE", "LhsDE", "OnePlusOne",
               "TwoPointsDE", "OnePointDE", "AlmostRotationInvariantDE", "RotationInvariantDE", "CMandAS2", "CMandAS"]
     if not parallel:
-        optims += ["SQP", "Cobyla", "Powell", "chainCMASQP", "chainCMAPowell"]
+        optims += ["SQP", "Powell", "chainCMASQP", "chainCMAPowell"]
+    if not parallel and not small:
+        optims += ["Cobyla"]
+    if not small:
+        optims += ["NGO", "Shiva"]
```
Comment (jrapin) [inline nevergrad/benchmark/experiments.py]: why remove them when it's small?

Comment (fteytaud) [reply nevergrad/benchmark/experiments.py]: Cobyla fails when the budget is too small ; I want to investigate it, but I won't have the time today, sorry.

Comment (teytaud) [reply nevergrad/benchmark/experiments.py]: After offline discussion: Jeremy refers to small budget and sequential. I try to investigate this, maybe NGO can be improved there. I also investigate large budgets.

Comment (fteytaud) [reply nevergrad/benchmark/experiments.py]: @jrapin I want to investigate :)

**Pull request merged**